source/manual/architecture-shallow-dive.html.md

```diff
@@ -0,0 +1,145 @@
+
+The request from Router is then proxied (or forwarded) to an application called Frontend.
+
+[Frontend](https://github.com/alphagov/frontend) is a public-facing app that serves the majority of documents on the /government part of GOV.UK. These government pages are fetched from the Content Store by the appropriate application. Frontend is a replacement for the public-facing parts of the Whitehall application.
+
+Frontend produces the homepage as an HTML response and sends that response back to Router.
+
+Router sends the response to Fastly, which caches the response for 30 minutes. This means that Fastly can handle any subsequent requests for the homepage during this time, without needing to query Origin at all.
+
+Finally, Fastly sends the response back to the user.
+
+### A user visits a GOV.UK webpage
+
+The user searches for a GOV.UK webpage, and selects the webpage link in the search results. For example, this [news page about changes to travel between Nigeria and England](https://www.gov.uk/government/news/uk-changes-travel-rules-for-travellers-from-nigeria-to-england).
+
+The user's browser uses [DNS](#dns) to figure out where to send the request, and sends that request to the Fastly content delivery network (CDN).
+
+If Fastly has the homepage cached, Fastly returns the homepage to the user.
+
+If Fastly does not have the homepage cached, Fastly sends the request on to Origin. Origin is the infrastructure that GOV.UK maintains, hosted in Amazon Web Services (AWS).
+
+Within Origin, the request goes to an application called Router to determine which GOV.UK application will produce the page.
+
+Router gets the request and decides which application should render the page. Router then proxies (forwards) the request to the Frontend application.
+
+Frontend looks at the request and decides which content to get from the Content Store. The [Content Store](https://github.com/alphagov/content-store) is:
+
+- an application that exposes the [Content Store API](https://github.com/alphagov/content-store/blob/main/docs/content-store-api.md)
+- a MongoDB database of almost all the content published on GOV.UK, excluding dynamic elements such as top links on taxon pages, navigation elements, or search result pages
+
+Frontend then queries the Content Store API to get the content in an [API response from the Content Store](https://www.gov.uk/api/content/government/news/uk-changes-travel-rules-for-travellers-from-nigeria-to-england).
+
+Frontend then builds the page using the content in the API response, and sends that response back to Router.
+
+Router sends the response to Fastly. Fastly sends the response back to the user.
+
+### A user searches GOV.UK using the search page
+
+The user goes to the [GOV.UK search page](https://www.gov.uk/search), and [searches for something, for example, "covid"](https://www.gov.uk/search/all?keywords=covid&order=relevance).
```

Comment:
Nitpick: I didn't consider this at the time, but we might want to pick a less sensitive / more timeless example, such as "tax". We'd just need to remember to update the URL above and the search.json URL later on.